src/components/CodeSnippet.tsx

```diff
@@ -63,7 +63,7 @@ export const CodeSnippet = memo(
                 variant="secondary"
                 startIcon="duplicate"
                 onClick={() => {
-                  copyToClipboard(code)
+                  copyToClipboard(code, true)
```

Comment:
Suggestion: maybe make the the option explicit.
Either name all attributes or make the last one part of an option object so we can
`copyToClipboard(code, { ignoreComment: true })` or something like that